tools/benchmark/cmd/lease.go

```diff
@@ -81,5 +81,6 @@ func leaseKeepaliveFunc(cmd *cobra.Command, args []string) {
 	wg.Wait()
 	close(r.Results())
 	bar.Finish()
-	fmt.Printf("%s", <-rc)
+	rv := <-rc
+	fmt.Printf("%s", rv.String())
```

Comment:
I expect `fmt.Printf("%s", <-rc)` just to work but Go somehow does not call the `String` method when used with ``fmt.Printf("%s", <-rc)``. So manually calls it.